### ISSUE TITLE
CAA record type is not included in this ansible module

### ISSUE THREAD

**Issue by mkearey** [ns1/ns1-ansible-modules#45]
Greetings,
May I ask why CAA record types were omitted from this module? I need it and can add it - unless there was a good reason not to?

Cheers

**Comment by mccarthy-james** [comment]
Hi. I don't believe there is a good reason that this is omitted. PRs are certainly welcome. Thank you!

**Comment by mkearey** [comment]
SOrry for the delay, I think I will have a go a this.

**Comment by michaelkearey** [comment]
Greetings I finally had an opportunity to attend to this.  Since I have no permissions in this project, I forked:
https://github.com/michaelkearey/ns1-ansible-modules

And then created a pull request:
https://github.com/ns1/ns1-ansible-modules/pull/47

I have also raised this issue with our customer success manager.

Note regarding tests/unit tests I have not created anything.  The tests/ns1_records.yml seems not to be fully functional as it is.
